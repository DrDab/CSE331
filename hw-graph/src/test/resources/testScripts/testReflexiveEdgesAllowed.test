# Create graphs w/ 1 and 2 nodes.
CreateGraph g1
AddNode g1 ALPHA

CreateGraph g2
AddNode g2 ALPHA
AddNode g2 BRAVO

# Add reflexive edge to graph with 1 node.
AddEdge g1 ALPHA ALPHA aa

# Add reflexive edge to graph with 2 nodes.
AddEdge g2 ALPHA ALPHA aa
AddEdge g2 BRAVO BRAVO bb