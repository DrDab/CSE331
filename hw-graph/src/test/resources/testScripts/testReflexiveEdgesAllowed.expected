# Create graphs w/ 1 and 2 nodes.
created graph g1
added node ALPHA to g1

created graph g2
added node ALPHA to g2
added node BRAVO to g2

# Add reflexive edge to graph with 1 node.
added edge aa from ALPHA to ALPHA in g1

# Add reflexive edge to graph with 2 nodes.
added edge aa from ALPHA to ALPHA in g2
added edge bb from BRAVO to BRAVO in g2